rules_python_gapic/py_gapic.bzl

```diff
@@ -0,0 +1,18 @@
+load("@com_google_api_codegen//rules_gapic:gapic.bzl", "proto_custom_library")
+
+def py_gapic_library2(name, srcs, **kwargs):
```

Comment:
Can we pass custom options through **kwargs? Is it straightforward?

Reply:
yes we can, we can also always add those custom options explicitly as args in this rule. It is straightforward to do (I did not add any because of my lack of knowledge and context here).

Reply:
Excellent. We will try using kwargs first and see if explicit args are necessary.